### ISSUE TITLE
Feature Request: Import classes

### ISSUE THREAD

**Issue by Zuldan** [nightroman/SplitPipeline#14]
@nightroman, thank you so much for putting this module together. I've used just about every other implementation of runspaces and this is by far the best. Best of all it works with PS 5.1.

I'd like to request the ability to import classes. Currently I'm having to duplicate the class inside of Split-Pipeline.

**Comment by nightroman** [comment]
I do not know how to do this. If this is possible at all. If you have ideas or some examples, please share.

**Comment by nightroman** [comment]
You may try to avoid code duplication by defining classes in shared and
imported functions or scripts.

Example

````powershell
    # my classes to be used by the current and parallel runspaces
    function Add-MyClass {
        class MyClass {
            [string]Test($message) { return "$(Get-Date) $message" }
        }
    }

    # some job using my classes
    function Test-Me($message) {
        $it = [MyClass]::new()
        $it.Test($message)
    }

    # try in the current runspace
    . Add-MyClass
    Test-Me current

    # try in parallel runspaces
    0..9 | Split-Pipeline -Function Add-MyClass, Test-Me -Begin {. Add-MyClass} {process{
        Test-Me "parallel $([runspace]::DefaultRunspace.InstanceId)"
    }}
````

**Comment by Zuldan** [comment]
@nightroman thanks. I will give this workaround a try.

**Comment by nightroman** [comment]
I am closing this. PowerShell does not provide supportive tools for this task. Feel free to reopen or create a new issue with some ideas or known solutions.

Here is a similar issue https://github.com/powercode/PSParallel/issues/2
